repository/src/main/java/org/alfresco/repo/content/replication/AggregatingContentStore.java

```diff
@@ -448,4 +442,97 @@ else if (!contentUrlSupported)
             readLock.unlock();
         }
     }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Experimental
+    @Override
+    public boolean requestSendContentToArchive(final String contentUrl)
+    {
+        return callContentArchiveRequest(contentUrl, Collections.emptyMap(), false);
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Experimental
+    @Override
+    public boolean requestRestoreContentFromArchive(final String contentUrl, final Map<String, Serializable> restoreParams)
+    {
+        return callContentArchiveRequest(contentUrl, restoreParams, true);
+    }
+
+    private boolean callContentArchiveRequest(final String contentUrl, final Map<String, Serializable> restoreParams, final boolean restore)
+    {
+        checkPrimaryStore();
+        // get a read lock so that we are sure that no replication is underway
+        readLock.lock();
+        boolean archiveRequestSucceeded = false;
+        boolean primaryContentUrlUnsupported = false;
+        boolean secondaryContentUrlUnsupported = false;
+        try {
```

Comment:
Formatting `{` new line
I can see we have 2 formatting styles in this file, some were not caught in previous PRs. 
This project looks like it used `{` on new lines.